samples/subsys/tracing/sample.yaml

```diff
@@ -8,6 +8,7 @@ common:
     regex:
       - "thread_a: Hello World from (.*)!"
       - "thread_b: Hello World from (.*)!"
+  extra_args: EXTRA_DTC_OVERLAY_FILE="app.overlay"
```

Comment:
could you please check if this can be moved to `sample.tracing.gpio` so it affects that test case only? It should look something like:

```yaml
    extra_args:
      - CONF_FILE=prj_gpio.conf
      - DTC_OVERLAY_FILE=gpio.overlay
```